lib/rules/typescript/prefer-pascal-case-enums.js

```diff
@@ -1,13 +1,13 @@
 const pascalCase = require('pascal-case');
+const {docsUrl} = require('../../utilities');
 
 module.exports = {
   meta: {
     docs: {
       description: 'Enforce Pascal case when naming enums.',
       category: 'Stylistic Issues',
       recommended: false,
-      uri:
-        'https://github.com/Shopify/eslint-plugin-shopify/blob/master/docs/rules/typescript/prefer-pascal-case-enums.md',
+      uri: docsUrl('prefer-pascal-case-enum'),
```

Comment:
Don't think this would generate the correct url. Previous one is coming from `.../rules/typescritpt/*.md`

Reply:
Ah good eyes! hoping this rule will catch those more easily!